test/unit/period/generators/bi-monthly_spec.js

```diff
@@ -14,11 +14,11 @@ describe('Bi-monthly period', () => {
 
     describe('generateBiMonthlyPeriodsForYear()', () => {
         it('should not allow years before the year zero', () => {
-            expect(() => generateBiMonthlyPeriodsForYear(-10)).to.throw('Generator does not support generating year before the year 0');
+            expect(() => generateBiMonthlyPeriodsForYear(-10)).to.throw();
```

Comment:
Yes I suppose the tests are less exact. They're also more brittle though since changing the error message would break the test. If the exception tests fail for a different reason than expected, then some of the success tests should fail as well.

In my opinion it's better if you only have to change existing tests when the functionality of the test subject changes, not cosmetic stuff like error messages. If you disagree, feel free to change it back :)